tests/__txcache__/testnet/56ebd82cdc91548617a1756f8e271dedda94e7c4d267d3c6d4f65d2654e5f3e2.json

```diff
@@ -0,0 +1,22 @@
+{
+  "bin_outputs": [
+    {
+      "amount": 10192086,
+      "script_pubkey": "00147a8e9720de35c5622b535bdbd1c1815632511b20"
+    },
+    {
+      "amount": 100000,
+      "script_pubkey": "0014e283f3e15f93b6dd27acba1f58b117ca96de3ff6"
+    }
+  ],
+  "inputs": [
+    {
+      "prev_hash": "9347a0298bd43f69ba168674f28bd9b25698db76f4f925582d9816eac2a61937",
+      "prev_index": 0,
+      "script_sig": "160014026813d5323be9c5d5f2df64a2ed4f02f5873669",
+      "sequence": 4294967294
+    }
+  ],
+  "lock_time": 1906598,
+  "version": 2
+}
```

Comment:
Can we add a NL pls? (applies to `tests/__txcache__/testnet/5bf10db6244c703d7f831043e6b6b0d6f3974bb5cf9fd216397e7620dc2e1015.json` as well)

Reply:
be2a8c1b8909f35c8fcc690d6299da036d26a914